frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts

```diff
@@ -177,8 +221,16 @@ export const enableYAMLValidation = (monaco, p2m, monacoURI, createDocument, yam
       .catch(() => {});
   };
 
+  let initialFoldingTriggered = false;
+
   getModel().onDidChangeContent(() => {
     const document = createDocument(getModel());
+
+    if (!initialFoldingTriggered && document.getText() !== '') {
+      fold(editor, monaco);
+      initialFoldingTriggered = true;
```

Comment:
It looks like the managed fields will get expanded again when the user clicks "Reload"